src/components/StorageOverview/Alerts/tests/__snapshots__/Alerts.test.js.snap

```diff
@@ -142,11 +259,90 @@ exports[`<Alerts /> renders correctly with Provider 1`] = `
   <div
```

Comment:
You could simplify this by using flex (which I think is already applied to `.kubevirt-dashboard__card-title`) and remove all of the bootstrap grid markup.

Reply:
@matthewcarleton  can you please specify what to simplify..sorry did not get you

Reply:
Sure! 
What happens when you remove the boostrap grid markup? Does it not get the right layout? 
`.kubevirt-dashboard__card-title` has the following css:
```
display: flex;
justify-content: space-between;
align-items: center;
```
so if you have the following markup:
```
<div class="card-pf-heading kubevirt-dashboard__card-heading">
  <h2 class="card-pf-title kubevirt-dashboard__card-title">
     Alerts
  </h2>
  <!--dropdown markup here :)-->
</div>
```
I would expect to see the layout you need.

Reply:
okay..so when I remove bootstrap grid I ma getting like this..which is not desired since dropdown is occupying all remaining space
![alerts](https://user-images.githubusercontent.com/25664409/57942093-3e0b7a80-78ee-11e9-954f-bcfebb3a01d5.png)

Reply:
ah ya
I don't think you'll need 
```<div class="btn-group btn-group-justified">```
wrapping it either.

Reply:
@matthewcarleton this is the snapshot file generated by snaphot tests. I cannot change it.

Reply:
ah ok my apologies for making this more confusing. I'm not very familiar with how this is built I am only judging the output. This may be out of scope for this issue, I do think it's worth revisiting though. We are using PF4 to layout these pages but then using PF3 bootstrap grid inside the cards. We can simplify it all quite a bit, I'm happy to help!